src/node_os.cc

```diff
@@ -87,29 +86,16 @@ static void GetOSType(const FunctionCallbackInfo<Value>& args) {
     return args.GetReturnValue().SetUndefined();
   }
 
-  args.GetReturnValue().Set(
-      String::NewFromUtf8(env->isolate(), info.sysname, NewStringType::kNormal)
-          .ToLocalChecked());
-}
-
-
-static void GetOSRelease(const FunctionCallbackInfo<Value>& args) {
-  Environment* env = Environment::GetCurrent(args);
-  uv_utsname_t info;
-  int err = uv_os_uname(&info);
-
-  if (err != 0) {
-    CHECK_GE(args.Length(), 1);
-    env->CollectUVExceptionInfo(args[args.Length() - 1], err, "uv_os_uname");
-    return args.GetReturnValue().SetUndefined();
-  }
+  // [sysname, version, release]
+  std::vector<Local<Value>> osInformation = {
+    FIXED_ONE_BYTE_STRING(env->isolate(), info.sysname),
+    FIXED_ONE_BYTE_STRING(env->isolate(), info.version),
+    FIXED_ONE_BYTE_STRING(env->isolate(), info.release)
```

Comment:
@addaleax This seems to be not working with utf8 strings, so the output it's quite weird, is there another helper for creating strings?

Reply:
```sh
 ./node -p "require('os').type()"
Darwin88Ñ×¿ïþÀJ@×¿ïþ8|ÿ8HØ¿ïþP¨²ÿ8p×¿ïþl8|ÿ8R% 
```

Reply:
@juanarbol So … you don’t want to use `FIXED_ONE_BYTE_STRING` because that assumes a fixed-size string; instead, you’ll probably want to use `String::NewFromUtf8` here (and leave the length unspecified)

Reply:
```cc
  // [sysname, version, release]
  Local<String> osInformation[] = {
    String::NewFromUtf8(env->isolate(), info.sysname),
    String::NewFromUtf8(env->isolate(), info.version),
    String::NewFromUtf8(env->isolate(), info.release)
  };

  args.GetReturnValue().Set(Array::New(env->isolate(), osInformation, arraysize(osInformation)));
```

This is not working, throws me this errors. Sorry that I'm asking you to resolve this issues, I'm quite new on v8:

```sh
../src/node_os.cc:91:5: error: no viable conversion from 'MaybeLocal<v8::String>' to 'Local<v8::String>'    String::NewFromUtf8(env->isolate(), info.sysname),    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~../deps/v8/include/v8.h:189:7: note: candidate constructor (the implicit copy constructor) not viable: no known conversion from 'MaybeLocal<v8::String>' to      'const v8::Local<v8::String> &' for 1st argumentclass Local {      ^../deps/v8/include/v8.h:189:7: note: candidate constructor (the implicit move constructor) not viable: no known conversion from 'MaybeLocal<v8::String>' to      'v8::Local<v8::String> &&' for 1st argument../deps/v8/include/v8.h:193:13: note: candidate template ignored: could not match 'Local' against 'MaybeLocal'  V8_INLINE Local(Local<S> that)            ^../src/node_os.cc:92:5: error: no viable conversion from 'MaybeLocal<v8::String>' to 'Local<v8::String>'    String::NewFromUtf8(env->isolate(), info.version),    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~../deps/v8/include/v8.h:189:7: note: candidate constructor (the implicit copy constructor) not viable: no known conversion from 'MaybeLocal<v8::String>' to      'const v8::Local<v8::String> &' for 1st argumentclass Local {      ^../deps/v8/include/v8.h:189:7: note: candidate constructor (the implicit move constructor) not viable: no known conversion from 'MaybeLocal<v8::String>' to      'v8::Local<v8::String> &&' for 1st argument../deps/v8/include/v8.h:193:13: note: candidate template ignored: could not match 'Local' against 'MaybeLocal'  V8_INLINE Local(Local<S> that)            ^../src/node_os.cc:93:5: error: no viable conversion from 'MaybeLocal<v8::String>' to 'Local<v8::String>'
    String::NewFromUtf8(env->isolate(), info.release)
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~../deps/v8/include/v8.h:189:7: note: candidate constructor (the implicit copy constructor) not viable: no known conversion from 'MaybeLocal<v8::String>' to      'const v8::Local<v8::String> &' for 1st argumentclass Local {      ^
../deps/v8/include/v8.h:189:7: note: candidate constructor (the implicit move constructor) not viable: no known conversion from 'MaybeLocal<v8::String>' to      'v8::Local<v8::String> &&' for 1st argument../deps/v8/include/v8.h:193:13: note: candidate template ignored: could not match 'Local' against 'MaybeLocal'  V8_INLINE Local(Local<S> that)
            ^
3 errors generated.make[1]: *** [/Users/juanjose/Documents/GitHub/Node/node/out/Release/obj.target/libnode/src/node_os.o] Error 1make: *** [node] Error 2
```

Reply:
`String::NewFromUtf8(env->isolate(), info.sysname).ToLocalChecked()`